src/main/java/org/jabref/Launcher.java

```diff
@@ -167,7 +168,7 @@ private static boolean handleMultipleAppInstances(String[] args, RemotePreferenc
                 if (remoteClient.sendCommandLineArguments(args)) {
                     // So we assume it's all taken care of, and quit.
                     LOGGER.debug("Arguments passed on to running JabRef instance.");
-                    System.out.println(Localization.lang("Arguments passed on to running JabRef instance. Shutting down."));
+                    LOGGER.info(Localization.lang("Arguments passed on to running JabRef instance. Shutting down."));
```

Comment:
Please revert - and add a comment that "System.out.println" is used for user output. (Line above is the logger - this line was really intended for a script-use output etc)